code/game/objects/items/storage/storage.dm

```diff
@@ -719,4 +746,39 @@
 	var/obj/item/drawn_item = contents[length(contents)]
 	drawn_item.attack_hand(user)
 
+///Called during Initialize this will iterate over spawns_with and spawns_prob to create our initial contents
 /obj/item/storage/proc/PopulateContents()
+	if(LAZYLEN(spawns_with))
+		var/iterations = 0
+		while(iterations < spawns_mult)
```

Comment:
meh.

rather than having spawns_mult, consider making the spawns_list an assoc list.

So it can be formatted like that:

spawns_list = (
/obj/item/whatever = 2,
/obj/item/something else = 1,
..
)

or like that 
spawns_list = (
/obj/item/whatever = 2,
/obj/item/something else,
..
)